ambassadors.html

```diff
@@ -61,9 +61,10 @@ <h1>Technica is calling on students from all around the country to work alongsid
           <div class="text-center">
             <p>As a Technica Campus Ambassador, you’ll have the unique opportunity to help foster our community beyond Technica weekend at UMD. As a representative of your school, you’ll spread the word about Technica 2020 on your campus and promote our message of inclusion in STEM for the largest all-women and non-binary hackathon in the world!</p>
           </div>
-          <div class="row">
-            <div class="col-md-4 description">
-              <h2 class="text-center">What do Campus Ambassadors do?</h2>
+          <a target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSdeSjURauxPmWzylOVq0sBjrWFAe0bWA_DCgyHmFqx_IdLH-Q/viewform"><img src="./img/ambassadors.jpg"></a>
```

Comment:
Overall, the page looks great! My only concern is that the image looks a bit odd because the top is cut off, can we increase the height via the top so it looks more natural?
![Screen Shot 2020-04-14 at 8 53 01 PM](https://user-images.githubusercontent.com/30019505/79287543-1c19cf80-7e92-11ea-89e9-aa4fc29441f2.png)